### PR TITLE
LatControlTorque: clean up class variable

### PR DESCRIPTION
**Pull request by ClockeNessMnstr**
just moving the self assignments to the superclass which is already called with CI/CP

### CODE REVIEW

**Comment by ClockeNessMnstr** [comment]
CI used but yea, self.CP and self.CI aren't used I believe. 

Could probably remove the self.CP = CP in torque
And remove CI from latcontrol.

File: selfdrive/controls/lib/latcontrol.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def __init__(self, CP, CI):
     self.sat_count_rate = 1.0 * DT_CTRL
     self.sat_limit = CP.steerLimitTimer
     self.sat_count = 0.
+    self.CP = CP
+    self.CI = CI
```
[the remark this answers is not in the document]
Comment (sshane) [reply selfdrive/controls/lib/latcontrol.py:15]: CI is used in LatControlPID and LatControlTorque in the init function, we don't need to make it a class variable

[the remark this answers is not in the document]
Comment (herman5) [reply selfdrive/controls/lib/latcontrol.py]: Let's remove `CI` from `LatControl` - no reason to have an used variable hanging around.

**Comment by sshane** [comment]
If you subclass, your init signature must be the same

**Comment by sshane** [comment]
If you subclass, your init signature must be the same. The angle subclass doesn't have an init function for example and controlsd always sends both CP and CI

**Pull request merged**